### PR TITLE
Meroxa builds describe

### PR DESCRIPTION
**Pull request by ericcheatham**
## Description of change
Fixes https://github.com/meroxa/turbine-project/issues/84

## Type of change

- [x]  New feature
- [ ]  Bug fix
- [ ]  Refactor
- [ ]  Documentation

## How was this tested?

- [x]  Unit Tests
- [ ]  Tested in staging

### CODE REVIEW

File: utils/display.go [hunk 1]
```diff
@@ -922,6 +922,62 @@ func AppTable(app *meroxa.Application) string {
...
+	}
+
+	table := simpletable.New()
+	if !hideHeaders {
+		table.Header = &simpletable.Header{
+			Cells: []*simpletable.Cell{
+				{Align: simpletable.AlignCenter, Text: "UUID"},
+				{Align: simpletable.AlignCenter, Text: "STATE"},
+				{Align: simpletable.AlignCenter, Text: "DETAILS"},
+				{Align: simpletable.AlignCenter, Text: "CREATED AT"},
```
Comment (janelletavares) [inline utils/display.go:937]: we tend to display both created at and updated at

File: utils/display.go [hunk 2]
```diff
@@ -922,6 +922,62 @@ func AppTable(app *meroxa.Application) string {
...
+			},
+		}
+	}
+
+	for _, p := range builds {
+		r := []*simpletable.Cell{
+			{Align: simpletable.AlignRight, Text: p.Uuid},
+			{Align: simpletable.AlignCenter, Text: p.Status.State},
+			{Align: simpletable.AlignCenter, Text: p.Status.Details},
+			{Align: simpletable.AlignCenter, Text: p.UpdatedAt},
```
Comment (janelletavares) [inline utils/display.go:947]: the heading is created, and the value is updated.  but i think we would want both anyway

File: cmd/meroxa/root/builds/builds.go [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+type Builds struct{}
+
+func (o *Builds) Usage() string {
+	return "builds"
+}
+
+func (o *Builds) Docs() builder.Docs {
+	return builder.Docs{
+		Short: "Manage builds on Meroxa",
```
Comment (janelletavares) [inline cmd/meroxa/root/builds/builds.go:37]:
```suggestion
		Short: "Manage Process builds on Meroxa",
```

File: cmd/meroxa/root/builds/describe.go [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+import (
+	"context"
+	"errors"
+
+	"github.com/meroxa/cli/utils"
+
+	"github.com/meroxa/cli/cmd/meroxa/builder"
+	"github.com/meroxa/cli/log"
+	"github.com/meroxa/meroxa-go/pkg/meroxa"
```
Comment (janelletavares) [inline cmd/meroxa/root/builds/describe.go:27]:
```suggestion
	"github.com/meroxa/cli/cmd/meroxa/builder"
	"github.com/meroxa/cli/log"
	"github.com/meroxa/cli/utils"
	"github.com/meroxa/meroxa-go/pkg/meroxa"
```

File: cmd/meroxa/root/builds/describe.go [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+	d.client = client
+}
+
+func (d *Describe) Logger(logger log.Logger) {
+	d.logger = logger
+}
+
+func (d *Describe) ParseArgs(args []string) error {
+	if len(args) < 1 {
+		return errors.New("requires build id")
```
Comment (janelletavares) [inline cmd/meroxa/root/builds/describe.go:83]:
```suggestion
		return errors.New("requires build UUID")
```

File: cmd/meroxa/root/builds/describe.go [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+var (
+	_ builder.CommandWithDocs    = (*Describe)(nil)
+	_ builder.CommandWithArgs    = (*Describe)(nil)
+	_ builder.CommandWithClient  = (*Describe)(nil)
+	_ builder.CommandWithLogger  = (*Describe)(nil)
+	_ builder.CommandWithExecute = (*Describe)(nil)
+)
+
+type describeBuildClient interface {
+	GetBuild(ctx context.Context, nameOrUUID string) (*meroxa.Build, error)
```
Comment (janelletavares) [inline cmd/meroxa/root/builds/describe.go:39]: this doesn't use the latest meroxa-go commit

File: cmd/meroxa/root/builds/describe_test.go [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+	"github.com/meroxa/meroxa-go/pkg/mock"
+)
+
+func TestDescribeBuildsArgs(t *testing.T) {
+	tests := []struct {
+		args []string
+		err  error
+		name string
+	}{
+		{args: nil, err: errors.New("requires build id"), name: ""},
```
Comment (janelletavares) [inline cmd/meroxa/root/builds/describe_test.go:40]:
```suggestion
		{args: nil, err: errors.New("requires build UUID"), name: ""},
```

File: cmd/meroxa/root/builds/logs.go [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+limitations under the License.
+*/
+
+package builds
+
+import (
+	"context"
+	"errors"
+
+	"io"
```
Comment (janelletavares) [inline cmd/meroxa/root/builds/logs.go:23]:
```suggestion
	"io"
```

File: cmd/meroxa/root/builds/describe_test.go [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+		{args: []string{"BuildId"}, err: nil, name: "BuildId"},
```
Comment (janelletavares) [inline cmd/meroxa/root/builds/describe_test.go:41]:
```suggestion
		{args: []string{"BuildUUID"}, err: nil, name: "BuildUUID"},
```

**Review by janelletavares — APPROVED** [review]

File: cmd/meroxa/root/builds/builds.go [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+	return true
+}
+
+func (*Builds) FeatureFlag() (string, error) {
+	return "turbine", fmt.Errorf("no access to the Meroxa Data Processes feature")
+}
+
+func (o *Builds) Docs() builder.Docs {
+	return builder.Docs{
+		Short: "Manage Process builds on Meroxa",
```
Comment (raulb) [inline cmd/meroxa/root/builds/builds.go:55]: Same here (it should be something like `Operate with Application builds on Meroxa`)`

File: cmd/meroxa/root/builds/describe.go [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+	}
+}
+
+func (d *Describe) Usage() string {
+	return "describe [UUID]"
+}
+
+func (d *Describe) Docs() builder.Docs {
+	return builder.Docs{
+		Short: "Describe a Meroxa Process Build",
```
Comment (raulb) [inline cmd/meroxa/root/builds/describe.go:56]: s/Process/Application

File: cmd/meroxa/root/builds/builds.go (hunk 10 above)
Comment (raulb) [reply cmd/meroxa/root/builds/builds.go:55]: Nvm, @janelletavares corrected me that this should be for Processes specifically. Ignore my comment.

File: utils/display.go [hunk 12]
```diff
@@ -960,6 +965,70 @@ func extendedFunctionsTable(functions []*meroxa.Function) string {
 	return subTable
 }
 
+// BuildsTable displays multiple build records for future listing per app.
+func BuildsTable(builds []*meroxa.Build, hideHeaders bool) string {
```
Comment (raulb) [inline utils/display.go:969]: I just noticed this is not being used anywhere. Will remove in https://github.com/meroxa/cli/pull/386 as part of a refactor I'm doing.

Comment (raulb) [reply utils/display.go:969]: Done in https://github.com/meroxa/cli/pull/386/commits/38ba1a0196473cb8ba0ad950e7f054803eda3b6b.

**Pull request merged**